### PR TITLE
Assets for 1.18

### PR DESCRIPTION
**Pull request by yabirgb**

### CODE REVIEW

**Comment by yabirgb** [comment]
Failing test is a timeout in etherscan when fetching eth2 deposits on chain. It also fails locally but I'm not sure if it's something in the etherscan side. If it keeps failing I'll investigate

File: rotkehlchen/tests/exchanges/test_kraken.py [hunk 1]
```diff
@@ -69,6 +69,8 @@ def test_coverage_of_kraken_balances(kraken):
     got_assets.remove('FLOWH.S')
     got_assets.remove('FLOWH')  # what is FLOWH?
     got_assets.remove('ADA.S')
+    got_assets.remove('SOL.S')
+    got_assets.remove('KSM.P')  # Couldn't find what this is
```
Comment (LefterisJP) [inline rotkehlchen/tests/exchanges/test_kraken.py:73]: I think I know! It's good to ignore like this for now. It's probably kusama bonded for parachains: 

https://www.kraken.com/en-gb/learn/parachain-auctions

Can you change the comment?

**Review by LefterisJP — COMMENTED** [review]
Some minor comments

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**